lib/fizzy/execute.cpp

```diff
@@ -176,6 +176,16 @@ Instance instantiate(const Module& module)
     // NOTE: fill it with zeroes
     bytes memory(memory_min * page_size, 0);
 
+    // Fill out memory based on data segments
+    for (auto const& data : module.datasec)
```

Comment:
```suggestion
    for (const auto& data : module.datasec)
```

Reply:
Which one is the preferred?

```
$ grep "const auto" * |wc -l
      48
$ grep "auto const" * |wc -l
      18
```

Reply:
I think it was mostly me who committed `auto const`, and it was a mistake.